source/_integrations/volkszaehler.markdown

```diff
@@ -14,45 +14,81 @@ related:
     title: Configuration file
 ---
 
-The **Volkszaehler** {% term integration %} is consuming the system information provided by the [Volkszaehler](https://wiki.volkszaehler.org/) API.
+The **Volkszaehler** {% term integration %} consumes the system information provided by the [Volkszaehler](https://wiki.volkszaehler.org/) API.
 
 ## Configuration
 
-To enable the Volkszaehler {% term integration %}, add the following lines to your {% term "`configuration.yaml`" %} file.
-{% include integrations/restart_ha_after_config_inclusion.md %}
+To enable the Volkszaehler {% term integration %}, use the Home Assistant user interface to configure the integration. This eliminates the need to manually edit the `configuration.yaml` file.
 
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: volkszaehler
-    uuid: DEVICE_UUID
-```
+### Step-by-Step Guide
+
+1. **Open Home Assistant**:
+   - Access your Home Assistant interface through your web browser.
```

Comment:
We have an generic re-usable include for this. Please check other integrations documentation page on how they do that.

We like to keep things standardized across all pages.

../Frenck